modules/avax-p-lib/package.json

```diff
@@ -0,0 +1,110 @@
+{
+  "name": "@bitgo/avax-p-lib",
```

Comment:
You'll need to add this package to the root `tsconfig.packages.json` file as well.

Reply:
updated!